demo.html

```diff
@@ -193,6 +198,7 @@
 
             return msgs[i];
         };
+
         var getMessageWithClearButton = function (msg) {
             msg = msg ? msg : 'Clear itself?';
             msg += '<br /><br /><button type="button" class="clear">Yes</button>';
```

Comment:
Add class "btn" to keep in line with Bootstrap styling used by other example toasts.